src/core/renderers/webgl/WebGLRenderer.js

```diff
@@ -440,18 +440,22 @@ export default class WebGLRenderer extends SystemRenderer
      * Changes the current shader to the one given in parameter
      *
      * @param {PIXI.Shader} shader - the new shader
+     * @param {boolean} [autoProject=true] - Whether automatically set the projection matrix
      * @return {PIXI.WebGLRenderer} Returns itself.
      */
-    bindShader(shader)
+    bindShader(shader, autoProject)
```

Comment:
That make sense, but because this is an anti-pattern, it's important that you leave a comment explaining that this is a performance optimization with Babel. In the future we may not use Babel for transpiling and would change this to be default params.